exe/floe

```diff
@@ -7,15 +7,32 @@ require "optimist"
 opts = Optimist.options do
   version "v#{Floe::VERSION}\n"
   opt :workflow, "Path to your workflow json", :type => :string, :required => true
-  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => '{}'
-  opt :credentials, "JSON payload with credentials", :type => :string, :default => '{}'
+  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => "{}"
+  opt :credentials, "JSON payload with credentials", :type => :string, :default => "{}"
+  opt :runner, "Type of runner for docker images", :type => :string, :default => "docker"
+  opt :runner_option, "Options to pass to the runner", :type => :strings
```

Comment:
Minor for readability:

```suggestion
  opt :inputs, "JSON payload to input to the workflow", :type => :string, :default => "{}"
  opt :credentials, "JSON payload with credentials",    :type => :string, :default => "{}"
  opt :runner, "Type of runner for docker images",      :type => :string, :default => "docker"
  opt :runner_option, "Options to pass to the runner",  :type => :strings
```

super-technical, but if you specify a default, you don't need to specify the type because it's inferred from the default.  This could also be:

```suggestion
  opt :inputs, "JSON payload to input to the workflow", :default => "{}"
  opt :credentials, "JSON payload with credentials",    :default => "{}"
  opt :runner, "Type of runner for docker images",      :default => "docker"
  opt :runner_option, "Options to pass to the runner",  :type => :strings
```